WalletWasabi.Gui/Global.cs

```diff
@@ -602,6 +605,12 @@ public async Task DisposeAsync()
 			{
 				await DisposeInWalletDependentServicesAsync();
 
+				if (HwiService != null)
+				{
+					await HwiService.StopAsync();
```

Comment:
This looks okay and I am sure it works flawlessly here, however **the pattern** is incorrect and it has been creating NRE problems. The reference to the property (in this case `HwiService`) has to be saved in a local variable first.

```c#
var hwiService = HwiService;
if (hwiService != null)
{
    await hwiService.StopAsync();
}
```

Reply:
Ack!